### PR TITLE
Add continuation support for comma and backslash characters.

### PR DESCRIPTION
**Pull request by marsupial**

### CODE REVIEW

File: test/Prompt/Continuation.C [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+CLING_MULTILINE_MACRO("DOWHILE");
+// CHECK-NEXT: DOWHILE
+
+#define CLING_MULTILINE_TRAILING_SPACE   \    
+  "Trailing Space "   \    
+  "And A Tab" \		
+  " End" // expected-warning@1 {{backslash and newline separated by space}} // expected-warning@2 {{backslash and newline separated by space}}
+
+// Separation by a tab is allowed? Is that a bug in clang?
```
Comment (Axel-Naumann) [inline test/Prompt/Continuation.C]: FYI and the record:
```
 echo '#define X \   
Y' | clang -x c++ -fsyntax-only -
<stdin>:1:12: warning: backslash and newline separated by space [-Wbackslash-newline-escape]
#define X \     
           ^
1 warning generated.
 echo '#define X \^V<TAB>
Y' | clang -x c++ -fsyntax-only -
<stdin>:1:12: warning: backslash and newline separated by space [-Wbackslash-newline-escape]
#define X \     
           ^
1 warning generated.
```
I.e. clang seems to warn about this.

Comment (marsupial) [inline test/Prompt/Continuation.C]: I'm not sure what you are trying to express...
If it a reference to the tab comment, this was written a while ago when compiling with 3.9 so they may have changed behavior.

Comment (Axel-Naumann) [inline test/Prompt/Continuation.C]: Thanks. I'll remove that comment then. Anyway - details; I'm waiting for ROOT's test suite to give the green light.

**Comment by Axel-Naumann** [comment]
Thanks, merged!

**Pull request closed without merging**